api/src/utils/jwt.ts

```diff
@@ -2,13 +2,14 @@ import jwt, { JsonWebTokenError, TokenExpiredError } from 'jsonwebtoken';
 import { DirectusTokenPayload } from '../types';
 import { InvalidTokenException, ServiceUnavailableException, TokenExpiredException } from '../exceptions';
 
-export function verifyAccessJWT(token: string, secret: string): DirectusTokenPayload {
+export function verifyJWT(token: string, secret: string): Record<string, any> {
 	let payload;
 
 	try {
-		payload = jwt.verify(token, secret, {
+		const { iat, iss, ...rest } = jwt.verify(token, secret, {
```

Comment:
@joselcvarela I think `iat` and `iss` should be left in the payload as it's a generic util function. Tests should be refactored instead, perhaps with `toMatchObject`. 🤔 

On a side note, there are some lint warnings.